src/devices/CharacterLcd/LcdInterface.cs

```diff
@@ -33,6 +33,12 @@ public abstract partial class LcdInterface : IDisposable
         /// <param name="values">Bytes to be send to the device</param>
         public abstract void SendData(ReadOnlySpan<byte> values);
 
+        /// <summary>
+        /// Sends data to the LCD device
+        /// </summary>
+        /// <param name="values">Bytes to be send to the device</param>
```

Comment:
Those are not really bytes... I don't find this very intuitive how you should implement this in the derived class? Should this instead be just some helper method like:

```csharp`
public void SendAscii(ReadOnlySpan<char> values)
{
    byte[] bytes = Encoding.ASCII.GetBytes(values);
    SendData(bytes);
}
```

Reply:
Will rephrase, it's actually char to be send. Good catch. Thanks.